devel/fetch_metrics/fetch_metrics.py

```diff
@@ -3,16 +3,16 @@
 import argparse
 import json
 import os
-from spikeextractors import RecordingExtractor, SortingExtractor
-from spikecomparison import GroundTruthComparison
-import spiketoolkit as st
 import time
-from typing import Any, Dict, List, TypedDict, Union
-
+from typing import Any, Dict, List, TypedDict, Union, cast
+import spikeextractors as se
```

Comment:
do not import spiketoolkit/spikecomparison here - in case we don't have those installed outside of the container. see below.